routes.go

```diff
@@ -30,15 +30,17 @@ type Server struct {
 	mux  *http.ServeMux
 	db   *DB
 	addr string
+	in   chan []byte
```

Comment:
This name is too generic and does not represent its purpose. Maybe this needs to be named `events`?